osbuild.py

```diff
@@ -201,27 +196,51 @@ def print_header(title, options, machine_name):
     print(f"\t# nsenter -a --wd=/root -t `machinectl show {machine_name} -p Leader --value`")
     print()
 
+class Pipeline:
```

Comment:
I agree that a class makes sense here.

All top-level things that get exported need to go into `__all__` (top of the file). Please add `Pipeline` and remove `run`.

Reply:
Fixed! Thanks!